bin/templates/cordova/lib/android_sdk.js

```diff
@@ -95,7 +94,7 @@ module.exports.list_targets = function () {
         } else throw err;
     }).then(function (targets) {
         if (targets.length === 0) {
-            return Q.reject(new Error('No android targets (SDKs) installed!'));
+            return Promise.reject(new Error('No android targets (SDKs) installed!'));
```

Comment:
I prefer to simply throw the Errors instead of wrapping them in a rejection and returning them. Might be personal taste though

Reply:
I have no strong feelings either way. But, when replacing `Q`, I wanted to keep the changes as simple as possible.

Reply:
Fair enough